### PR TITLE
SIG: initial commit for planner sig

### PR DESCRIPTION
**Pull request by winoros**
initial commit for planner sig

### CODE REVIEW

File: special-interest-groups/sig-planner/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: [ttps://pingcap.zoom.com.cn/j/492702291](ttps://pingcap.zoom.com.cn/j/492702291)
+* Meeting Notes: to be updated
+
+## Contact
+
+* Slack: channel #sig-planner in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (gingerkidney) [inline special-interest-groups/sig-planner/README.md]:
```suggestion
* Slack: channel #sig-planner in the [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```

File: special-interest-groups/sig-planner/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+See [Planner SIG Roles and Organization Management](./roles-and-organization-management.md)
+
+## Members
+
+See [SIG Planner Member List](./member-list.md)
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (gingerkidney) [inline special-interest-groups/sig-planner/README.md]: 5:00 AM or 5:00 PM?

File: special-interest-groups/sig-planner/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Contact
+
+* Slack: channel #sig-planner in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
+
+## Code Locations
+
+* The [planner package](https://github.com/pingcap/tidb/tree/master/planner) in the tidb repo.
+* The [statistics package](https://github.com/pingcap/tidb/tree/master/statistics) in the tidb repo.
+* Other codes that affect the correctness, performance of the planner, like say the [expresion package](https://github.com/pingcap/tidb/tree/master/expression).
```
Comment (gingerkidney) [inline special-interest-groups/sig-planner/README.md]:
```suggestion
* Other codes that affect the correctness, performance of the planner, like say the [expression package](https://github.com/pingcap/tidb/tree/master/expression).
```

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+the **Active Contributor** community role in the planner/statistics package.
+
+### Promotion
+
+#### From `Contributor` to `Active Contributor`
+
+Requirements:
+
+* Have at least 3 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Planner SIG Reviewers, Committer or Maintainer.
```
Comment (gingerkidney) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]:
```suggestion
* Nominated by at least 2 Planner SIG Reviewers or role above.
```

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* Have at least 10 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Committers or 2 Maintainers.
+
+#### From `Reviewer` to `Committer`
+
+Requirements:
+
+* Have at least 30 PRs merged in the planner/statistics package within one year.
+* Have reviewed at least 15 PRs in the planner/statistics package within one year.
+* Nominated by at least 2 Committers or Maintainers.
```
Comment (gingerkidney) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]: The promotion for reviewer to committer should not conflict with the [TiKV community membership](https://github.com/tikv/community/blob/master/community-membership.md), committer must have approval by 2 maintainers or above

File: special-interest-groups/sig-planner/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Planner Special Interest Group(Planner SIG)
+
+Planner SIG covers TiDB planner/statistics implementations. We are focused on
+the development and maintenance of the TiDB [planner](https://pingcap.com/docs/stable/reference/performance/sql-optimizer-overview/)
+and [statistics](https://pingcap.com/docs/stable/reference/performance/statistics/) component, including but not limited to:
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/README.md]:
```suggestion
and [statistics](https://pingcap.com/docs/stable/reference/performance/statistics/) components, including but not limited to:
```

Comment (zz-jason) [reply special-interest-groups/sig-planner/README.md]: You can also describe the mission of TiDB Planner

File: special-interest-groups/sig-planner/README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## Members
+
+See [SIG Planner Member List](./member-list.md)
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: [ttps://pingcap.zoom.com.cn/j/492702291](ttps://pingcap.zoom.com.cn/j/492702291)
+* Meeting Notes: to be updated
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/README.md]: please add a public meeting notes doc.

File: special-interest-groups/sig-planner/member-list.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2019-11-24
+
+## Active Contributors
+
+Members and their contributions:
+
+None
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/member-list.md]: I think we may already have some, please check it carefully.

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+Requirements:
+
+* Have at least 3 PRs merged in the planner/statistics package within one year.
+* Nominated by at least 2 Planner SIG Reviewers, Committer or Maintainer.
+
+#### From `Active Contributor` to `Reviewer`
+
+Requirements:
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]: How about 20?

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+* Nominated by at least 2 Committers or 2 Maintainers.
+
+#### From `Reviewer` to `Committer`
+
+Requirements:
+
+* Have at least 30 PRs merged in the planner/statistics package within one year.
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]: how about 40 or 50?

File: special-interest-groups/sig-planner/roles-and-organization-management.md (hunk 5 above)
Comment (winoros) [reply special-interest-groups/sig-planner/roles-and-organization-management.md]: It's copied from sig-expr. So i think @qw4990 should fix that one too?

File: special-interest-groups/sig-planner/roles-and-organization-management.md (hunk 10 above)
Comment (winoros) [reply special-interest-groups/sig-planner/roles-and-organization-management.md]: Add one more requirement that there must be enough valuable issues fixed by him/her.

File: special-interest-groups/sig-planner/README.md (hunk 7 above)
Comment (winoros) [reply special-interest-groups/sig-planner/README.md]: Do we need to fill it before first meeting is held?

File: special-interest-groups/sig-planner/member-list.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
+# SIG Planner Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2019-11-24
```
Comment (winoros) [inline special-interest-groups/sig-planner/member-list.md]: Do we need to maintain this field?
The update history is maintained by github too.

File: special-interest-groups/sig-planner/README.md (hunk 2 above)
Comment (winoros) [reply special-interest-groups/sig-planner/README.md]: AM. can be checked by that link

File: special-interest-groups/sig-planner/roles-and-organization-management.md (hunk 9 above)
Comment (winoros) [reply special-interest-groups/sig-planner/roles-and-organization-management.md]: One more requirement about the number of valuable issues too.

File: special-interest-groups/sig-planner/README.md (hunk 7 above)
Comment (gingerkidney) [reply special-interest-groups/sig-planner/README.md]: I think a meeting note link is ok, even there is no content for now.

File: special-interest-groups/sig-planner/member-list.md (hunk 8 above)
Comment (winoros) [reply special-interest-groups/sig-planner/member-list.md]: @Reminiscent and @SeaRise is going to reach the requirements soon.
Currently is none.

Comment (winoros) [reply special-interest-groups/sig-planner/member-list.md]: Update: they still need to waiting for current pr merged.

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+* Have at least 10 PRs merged in the planner/statistics package within one year.
+* Fix at least 2 issues whose difficulty is medium or above.
+
+#### Be promoted to [`Committer`](../../architecture/README.md#committer)
+
+SIG Specific Requirements:
+
+* Have at least 20 PRs merged in the planner/statistics package within one year.
+* Fix at least 10 issues whose difficulty is medium or above.
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]: Can we reduce the number of solved medium issues? It's too high, which makes a huge gap between `reviewer` and `committer` 😂

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+**How to rejoin**:
+
+* Should meet the requirements for Active Contributor again.
+
+### Reviewer
+
+**Automatically demoted**:
+
+For a Reviewer to be demoted to an Active Contributor,  he/she must satisfy at
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]:
```suggestion
For a Reviewer to be demoted to an Active Contributor, he/she must satisfy at
```

File: special-interest-groups/sig-planner/roles-and-organization-management.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+* Haven't appeared in the SIG meeting without any reason for 3 times continuously.
+* At least 2 Committers or Maintainers agree that the Reviewer is not qualified.
+
+**How to retain**:
+
+* Have reviewed or merged 5 PRs within 1 month.
+
+### Voluntarily Quit or Demote
+
+Please email to [Cui Yiding](mailto:cuiyiding@pingcap.com) if you are
```
Comment (zz-jason) [inline special-interest-groups/sig-planner/roles-and-organization-management.md]: please use the email of Yanqing

**Pull request merged**